### DISCUSSION TITLE
Call API and get: Request Rejected

### DISCUSSION THREAD

**Discussion by orrmendelson** [NREL/REopt-API-Analysis#156] [8 - API: Application Programming Interface]
Hi,

I tested the API and passed the KEY verification stage, but got:
"Request Rejected, An error has occurred. Please consult with your administrator. Your support ID is: 1779917886979535304.

What should I do?

**Comment by LParkhill** [comment]
Hi, 

Unfortunately, this isn't a REopt error message, so the REopt Team can't track the problem. If you are still encountering it, please contact NREL Developer Network. There have been other users who have posted "Request Rejected" issues and have had them resolved.   https://github.com/NREL/developer.nrel.gov/issues

Thanks
